charts/ethereum/values.yaml

```diff
@@ -0,0 +1,286 @@
+terminalTotalDifficulty: ""
+# Defines whether the Merge plugin is enabled bundles are allowed
+mergeEnabled: true
+# URL to Builder Relay. If set when building blocks client will send them to the relay.
+builderEndpoint: ""
+# Post bellatrix, this address will receive the transaction fees produced by any blocks
+# from this node. Default to junk whilst bellatrix is in development state.
+# Validator client can override this value through the preparebeaconproposer api.
+# (default: "0x0000000000000000000000000000000000000000")
+suggestedFeeRecipient: ""
+
+# Consensus Client RPC Settings
+consensusClientRpcPort: 8555
+# Execution Client RPC Settings
+executionClientRpcPort: 8555
+# API's offered over the HTTP-RPC interface
+executionClientApiModules: "web3,eth,net,engine"
+
+# Set the frequency (in slots) at which to store finalized states to disk.
+# Specifying a larger number of slots as the archive frequency has a potentially higher
+# overhead for retrieving finalized states since more states may need to be regenerated
+# to get to the requested state.
+# Specifying a lower number of slots as the frequency increases the disk space usage.
+dataStorageArchiveFrequency: "8192"
+
+
+# Extra flags to pass to the node
+extraFlags:
+  executionClient: []
+  consensusClient: []
+
+imageRepository:
+  geth: ethereum/client-go
+  nethermind: nethermind/nethermind
+  erigon: thorax/erigon
+  prysm: gcr.io/prysmaticlabs/prysm/beacon-chain
+  prysmGnosis: ghcr.io/gnosischain/gbc-prysm-beacon-chain
+  lighthouse: sigp/lighthouse
+  teku: consensys/teku
+imageTag:
+  geth: v1.10.21
+  nethermind: 1.13.4
+  erigon: v2022.07.03
+  prysm: v2.1.4-rc.0
+  prysmGnosis: v2.1.2-gbc
+  lighthouse: v2.5.0
+  teku: 22.7.0
+imagePullPolicy: IfNotPresent
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Init image is used to chown data volume, initialise genesis, etc.
+initImage:
+  repository: "busybox"
+  tag: "1.35"
+  pullPolicy: IfNotPresent
+
+# Sidecar image is used to perform Liveness/Readiness probes.
+sidecar:
+  repository: "europe-west4-docker.pkg.dev/stakewiselabs/public/ethnode-sidecar"
+  tag: "v1.0.3"
+  pullPolicy: IfNotPresent
+  bindAddr: "0.0.0.0"
+  bindPort: 3000
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations:
+  geth: {}
+  nethermind: {}
+  erigon: {}
+  prysm: {}
+  lighthouse: {}
+  teku: {}
+
+securityContext:
+  geth:
+    fsGroup: 1001
+    runAsUser: 1001
+  nethermind:
+    fsGroup: 1000
+    runAsUser: 1000
+  erigon:
+    fsGroup: 1001
+    runAsUser: 1001
+  prysm:
+    fsGroup: 1001
+    runAsUser: 1001
+  lighthouse:
+    fsGroup: 1001
+    runAsUser: 1001
+  teku:
+    fsGroup: 1000
```

Comment:
Why teku has a different values here compared to others?

Reply:
It comes from upstream image, under "1000" user process failed to start